metricbeat/mb/module/wrapper.go

```diff
@@ -106,9 +106,17 @@ func (mw *Wrapper) Start(done <-chan struct{}) <-chan beat.Event {
 	wg.Add(len(mw.metricSets))
 	for _, msw := range mw.metricSets {
 		go func(msw *metricSetWrapper) {
+			metricsPath := fmt.Sprintf("%s", msw.ID())
+			registry := monitoring.GetNamespace("dataset").GetRegistry()
```

Comment:
Long term this will the case. I didn't change it in this PR as I didn't want to change any of the existing stats outputs as it could break central monitoring.